spec/services/bulk_submission_service_spec.rb

```diff
@@ -78,9 +85,56 @@
           ),
         ])
       end
+
+      it "updates bulk_submission status to \"prepared\"" do
+        expect{ call }
+          .to change { bulk_submission.reload.status }
+            .from("pending")
+            .to("prepared")
+      end
+
+      it "enqueues HmrcInterfaceBulkSubmissionWorker job for bulk_submission", type: :worker do
+        expect { call }
+          .to change(HmrcInterfaceBulkSubmissionWorker, :jobs)
+          .from([])
+          .to(
+            [
+              hash_including(
+                "retry" => true,
```

Comment:
🤔 perhaps we should be limiting this to 5 or some number lower than that for dependant workers